recipes/imguizmo/all/conanfile.py

```diff
@@ -42,20 +43,24 @@ def layout(self):
 
     def requirements(self):
         # 1.89 is the newest imgui version compatible with imguizmo
-        self.requires("imgui/1.89.3", transitive_headers=True)
+        self.requires("imgui/1.90.4", transitive_headers=True)
 
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
 
     def generate(self):
         tc = CMakeToolchain(self)
+        tc.preprocessor_definitions["IMGUI_DEFINE_MATH_OPERATORS"] = ""
         tc.generate()
         tc = CMakeDeps(self)
         tc.generate()
 
     def build(self):
+        if Version(self.dependencies["imgui"].ref.version) >= "1.89.4":
+            replace_in_file(self, os.path.join(self.source_folder, "GraphEditor.cpp"),
+                            "AddBezierCurve", "AddBezierCubic")
```

Comment:
[checking the changelog looks like this is only a redirect](https://github.com/ocornut/imgui/blob/4d8e839ddf3786f11390002b86218419311843fb/docs/CHANGELOG.txt#L42), so I think this patch is fine.